synapse/lib/cell.py

```diff
@@ -809,6 +810,14 @@ async def getGcInfo(self):
             'threshold': gc.get_threshold(),
         }
 
+    @adminapi()
```

Comment:
Is having multiple named reload targets worth the complexity vs frequency of use? Is there harm in reloading all-the-things?

Reply:
I didn't want to preclude the option up front for a more complex user. If we removed it now we could always add it back if needed.

Reply:
Sounds fine, just checking. :+1: